CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## v0.0.3
+
+*September 1, 2020*
+
+This release focuses on the IBC message processor framework (formerly known as handler framework) and initial implementations in ICS02 and ICS03. It also introduces an initial specification for the relayer algorithm.
+
+Other highlights:
+- A new `ibc` crate is released from the previous `modules` crate.
+- ADR-001 is complete. 🎉
+
+### BREAKING CHANGES:
+
+- [modules] Renamed `modules` crate to `ibc` crate. Version number for the new crate is not reset. ([#198])
+- [modules/ics02] `ConnectionId`s are now decoded to `Vec<ConnectionId>` and validated instead of `Vec<String>` ([#185])
+- [modules/ics03] Removed `Connection` and `ConnectionCounterparty` traits ([#193])
+- [modules/ics04] Removed `Channel` and `ChannelCounterparty` traits ([#192])
+
+### FEATURES:
+- [modules/ics02] partial implementation of message handler ([#119], [#194])
+- [modules/ics03] partial implementation of message handler ([#119], [#194])
+- [architecture/ADR-003] Proposal for IBC handler architecture ([#119], [#194])
```

Comment:
```suggestion
- [architecture/ADR-003] Proposal for IBC handler (message processor) architecture ([#119], [#194])
```

Earlier in this changelog we called it "message processor framework". We should be consistent..